README.md

```diff
@@ -14,12 +14,20 @@ Spotipy's full documentation is online at [Spotipy Documentation](http://spotipy
 pip install spotipy
 ```
 
+alternatively 
+
+```bash
+py -m pip install spotipy
```

Comment:
what is `py`?

Reply:
Py is the python launcher for windows, where python is the python executable. I could not install Spotipy using python.